package com.Exception;

public class Test {
    public static void main(String[] args) {
//        User u = new User("w", "123");
//        Login l = new Login();
//        l.login(u);
////        try {
////
////        } catch (UsernameNotFoundException | PasswordNotMatchException e) {
////            System.out.println(e.getMessage());
////        }
        Integer i = 10;
        int j = 10;
        System.out.println(i.equals(10));

    }
}
